### PR TITLE
Updating Training Compiler Single Node Multi GPU notebook to use HF-PT 1.11 

### PR DESCRIPTION
**Pull request by Lokiiiiii**
*Description of changes:*
This PR is part of a set to update Training Compiler notebooks with our latest release for PT 1.11

*Testing done:*
Local testing completed in SageMaker Notebook: 
[language-modeling-multi-gpu-single-node.pdf](https://github.com/aws/amazon-sagemaker-examples/files/9530326/language-modeling-multi-gpu-single-node.pdf)

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by atqy — DISMISSED** [review]

File: sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling/gpt-2.ipynb [hunk 1]
```diff
@@ -2,15 +2,15 @@
  "cells": [
```
Comment (mchoi8739) [inline sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling/gpt-2.ipynb:2]: <p>I haven't been informed that the native PyTorch support became available. We added the new <code>pytorchxla</code> as distribution strategy only for the HugginigFace estimator. That's also how we documented in the <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/training-compiler-enable-pytorch.html" target="_blank">dev guide</a> and the <a href="https://sagemaker.readthedocs.io/en/stable/frameworks/huggingface/sagemaker.huggingface.html#hugging-face-estimator" target="_blank">SageMaker HuggingFace estimator's docstring</a>. We didn't added <code>pytorchxla</code> to the <a href="https://sagemaker.readthedocs.io/en/stable/frameworks/pytorch/sagemaker.pytorch.html" target="_blank">SageMaker PyTorch estimator</a>.</p><p> </p><p>Is this statement true?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3593/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (Lokiiiiii) [reply sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling/gpt-2.ipynb:2]: Yes, pytorchxla support is restricted to HuggingFace estimator with the Training Compiler enabled. On an unrelated note, pytorchddp support is currently only available for the PyTorch estimator.

Training compiler does not support PyTorch estimator.

Comment (mchoi8739) [reply sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling/gpt-2.ipynb:2]: Then the text here should be fixed/reverted

Comment (Lokiiiiii) [reply sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling/gpt-2.ipynb:2]: <p>The text already reflects the use of PyTorch estimator for native and HuggingFace estimator with training compiler. Can you make a suggestion ?</p>

Comment (mchoi8739) [reply sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling/gpt-2.ipynb:2]: <p>I think the statement "We use a <code>PyTorch</code> estimator for native PyTorch" is wrong for any case.</p>

Comment (mchoi8739) [reply sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling/gpt-2.ipynb:2]: <p>Even without the distribution strategy pytorchxla, we do not support the native PyTorch, as far as I understand.</p><p> </p>

Comment (mchoi8739) [reply sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling/gpt-2.ipynb:2]: <p>The SageMaker PyTorch estimator class does not have the arg for SMTC. https://sagemaker.readthedocs.io/en/stable/frameworks/pytorch/sagemaker.pytorch.html</p>

Comment (mchoi8739) [reply sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling/gpt-2.ipynb:2]: <p>Currently we support native TF through the SM TensorFlow estimator, HF-PT&amp;HF-TF through the SM HuggingFace estimator. But not the native PyTorch.</p>

**Review by mchoi8739 — APPROVED** [review]

**Review by atqy — APPROVED** [review]

**Pull request merged**